### ISSUE TITLE
[Ubuntu] Default Clang will be set to Clang 11 on December, 14

### ISSUE THREAD

**Issue by AlenaSviridenko** [actions/virtual-environments#2130]
**Breaking changes**
Default Clang will be set to Clang 11 on Ubuntu 20.04 image (PR https://github.com/actions/virtual-environments/pull/2087).

**Target date**
Image deployment will start on ~December, 7~ December, 14 and will take 2-3 days

**The motivation for the changes**
We'd like to provide the image with the latest actual software provided by default.

**Possible impact**
If your project is not compatible with Clang 11 and you are using default Clang - your build will be broken.

**Virtual environments affected**
- [ ] Ubuntu 16.04
- [ ] Ubuntu 18.04
- [x] Ubuntu 20.04
- [ ] macOS 10.13
- [ ] macOS 10.14
- [ ] macOS 10.15
- [ ] macOS 11.0
- [ ] Windows Server 2016 R2
- [ ] Windows Server 2019

**Mitigation ways**
Run the following script to set Clang 10 as a default version explicitly:
```
sudo update-alternatives --remove-all clang
sudo update-alternatives --install /usr/bin/clang++ clang++ /usr/bin/clang++-10 100
sudo update-alternatives --install /usr/bin/clang clang /usr/bin/clang-10 100
sudo update-alternatives --install /usr/bin/clang-format clang-format /usr/bin/clang-format-10 100
```

**Comment by maxim-lobanov** [comment]
We have to postpone this update for now since Clang 11 is not available for Ubuntu 20.04 in official apt repo yet.
Will revisit it later.

**Comment by miketimofeev** [comment]
@dmitryikh thanks for the update! Could you please create an issue if you need clang11 on the image?
